usbhost.h

```diff
@@ -57,6 +57,10 @@ template< typename SPI_CLK, typename SPI_MOSI, typename SPI_MISO, typename SPI_S
         static void init() {
                 // Should be initialized by the user manually for now
         }
+#elif defined(AM_PART_APOLLO3)
+		static void init() {
+				SPI::begin();
+		}
```

Comment:
Please remove this.

Reply:
Instead add the following to the bottom of [settings.h](https://github.com/felis/USB_Host_Shield_2.0/blob/b56e8b76a1bf1b93be8c82232e34c9c17d799bbd/settings.h#L217):

```C++
// See: https://github.com/sparkfun/Arduino_Apollo3/pull/421
#ifdef AM_PART_APOLLO3
#ifndef SPI_HAS_TRANSACTION
#define SPI_HAS_TRANSACTION 1
#endif
#endif
```

Reply:
removed, and updated